api-put-object-streaming.go

```diff
@@ -56,13 +56,13 @@ func (c Client) putObjectMultipartStream(ctx context.Context, bucketName, object
 		if errResp.Code == "AccessDenied" && strings.Contains(errResp.Message, "Access Denied") {
 			// Verify if size of reader is greater than '5GiB'.
 			if size > maxSinglePutObjectSize {
-				return 0, errEntityTooLarge(size, maxSinglePutObjectSize, bucketName, objectName)
+				return UploadInfo{}, errEntityTooLarge(size, maxSinglePutObjectSize, bucketName, objectName)
 			}
 			// Fall back to uploading as single PutObject operation.
 			return c.putObject(ctx, bucketName, objectName, reader, size, opts)
 		}
 	}
-	return n, err
+	return info, err
```

Comment:
UploadInfo struct does not have size - whereas ObjectInfo set both size and ETag previously in PutObject. Think we should continue to have size in the returned struct as applications may be using this today.

Reply:
There is more stuff missing,  please join #minio-go-v7 for more information

Reply:
> UploadInfo struct does not have size - whereas ObjectInfo set both size and ETag previously in PutObject. Think we should continue to have size in the returned struct as applications may be using this today.

Done